server/src/main/java/org/elasticsearch/index/seqno/RetentionLeaseBackgroundSyncAction.java

```diff
@@ -81,7 +81,9 @@ public RetentionLeaseBackgroundSyncAction(
             actionFilters,
             Request::new,
             Request::new,
-            threadPool.executor(ThreadPool.Names.MANAGEMENT)
+            threadPool.executor(ThreadPool.Names.MANAGEMENT),
+            true,
```

Comment:
I don't think we should be changing `syncGlobalCheckpointAfterOperation` to `true`.